### PR TITLE
IDVA5-1145- enum values in DB fix 

### PR DESCRIPTION
**Pull request by atedros-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

### Type of change

* [ ] Bug fix
* [ ] New feature
* [ ] Breaking change
* [ ] Infrastructure change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [ ] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/acsp/models/enums/RoleType.java [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
-    MEMBER_OF_LLP("Member of LLP"),
-    GENERAL_PARTNER("General partner"),
-    SOMEONE_ELSE("Someone else");
+    SOLE_TRADER("sole-trader"),
+    MEMBER_OF_PARTNERSHIP("member-of-partnership"),
+    MEMBER_OF_GOVERNING_BODY("member-of-the-governing-body"),
+    EQUIVALENT_OF_DIRECTOR("equivalent-to-director"),
+    MEMBER_OF_ENTITY("member-of-the-entity"),
+    DIRECTOR("director"),
+    MEMBER_OF_LLP("member-of-LLP"),
```
Comment (sampankumar) [inline src/main/java/uk/gov/companieshouse/acsp/models/enums/RoleType.java:10]: Make LLP as lower case as well please.

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sampankumar — APPROVED** [review]

**Review by sardhani-ch — APPROVED** [review]

**Review by cbroadley-ch — APPROVED** [review]

**Pull request merged**